doc/varnish-configuration.rst

```diff
@@ -10,11 +10,11 @@ and 4.
 Basic Varnish Configuration
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To invalidate cached objects in Varnish, begin by adding an
-`ACL <https://www.varnish-cache.org/docs/3.0/tutorial/vcl.html#example-3-acls>`_
-to your Varnish configuration. This ACL determines which IPs are allowed to
-issue invalidation requests. Let’s call the ACL `invalidators`. The ACL below
-will be used throughout the Varnish examples on this page.
+To invalidate cached objects in Varnish, begin by adding an `ACL` (for Varnish
+3 see `ACL for Varnish 3`_) to your Varnish configuration. This ACL determines
```

Comment:
there is no difference actually. i just wanted to link to the proper version of the documentation. not sure how we should handle that. duplicating the doc config with no difference seems a bit strange. should i just say that this did not change?